jenkins/e2e-image/e2e-runner.sh

```diff
@@ -176,6 +176,11 @@ function install_google_cloud_sdk_tarball() {
     mkdir -p "${install_dir}"
     tar xzf "${tarball}" -C "${install_dir}"
 
+    if running_in_docker; then
+        # Delete any previously installed SDK to prevent weirdly overlapping
+        # installs from fighting each other.
+        rm -rf "${install_dir}/google-cloud-sdk"
+    fi
```

Comment:
Yeah, that would be a better solution, but it might break something else. Erick did the change that added it to the image, and I think *some* jobs rely on it being there.